src/LessMsi.Gui/MainForm.cs

```diff
@@ -899,13 +899,20 @@ private void UpdateControlLayout(Panel panel)
 
         private void OpenFileCommand()
         {
-            if (DialogResult.OK != openMsiDialog.ShowDialog(this))
+            DialogResult dialogResult = openMsiDialog.ShowDialog(this);
+
+            if (DialogResult.OK == dialogResult)
             {
-                Presenter.Error(string.Format(Strings.FileExistError, openMsiDialog.FileName));
-                txtMsiFileName.Text = SelectedMsiFileFullName;
-                return;
+                if (File.Exists(openMsiDialog.FileName))
+                {
+                    LoadFile(openMsiDialog.FileName);
+                }
+                else
+                {
+                    Presenter.Error(string.Format(Strings.FileExistError, openMsiDialog.FileName));
+                    txtMsiFileName.Text = SelectedMsiFileFullName;
+                }

```

Comment:
Inverting this first if statement will reduce the cyclomatic complexity of this method. It's super minor in this case, but I've found that in larger methods it can help and if applied throughout the codebase it helps reduce overall complexity of the codebase. 

```suggestion
            if (DialogResult.OK == dialogResult)
                return;

            if (File.Exists(openMsiDialog.FileName))
            {
                LoadFile(openMsiDialog.FileName);
                return;
            }

            Presenter.Error(string.Format(Strings.FileExistError, openMsiDialog.FileName));
            txtMsiFileName.Text = SelectedMsiFileFullName;
```